assignment/scripts/function-practice.js

```diff
@@ -12,54 +12,64 @@ function hello() {
 console.log('Test - should say "Hello World!"', hello());
 
 
-// 2. Function to return an personalized hello, using the `name` argument.
+// 2. Function to return a personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
 function helloName( name ) {
-  return;
+  return 'Hello, Dolly!';
 }
 // Remember to call the function to test
-
+console.log(helloName());
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
-  // return firstNumber + secondNumber;
+function addNumbers( firstNumber , secondNumber ) {
+let addNumbers = firstNumber + secondNumber;
+  return addNumbers;
 }
-
+console.log(addNumbers ( 5, 7 ));
 
 // 4. Function to multiply three numbers & return the result
-function multiplyThree( ){
-
+function multiplyThree( numberOne , numberTwo, numberThree) {
+let multiplyThree = numberOne * numberTwo * numberThree;
+  return multiplyThree;
 }
-
+console.log (multiplyThree (2, 3, 4 ));
 
 // 5. Function that will return true if a number is positive, 
 //    or greater than zero, and false otherwise
 function isPositive( number ) {
   if ( number > 0 ){
-    return;
+    return true;
+  }
+  else {
+    return false;
   }
-    return;
 }
 // Call the function to test each outcome (true & false) 
 // Write a separate console.log statement for each outcome
-console.log( 'isPositive - should say true', isPositive(3) );
-console.log( 'isPositive - should say false', isPositive(0) );
-console.log( 'isPositive - should say false', isPositive(-3) );
+console.log( 'The number 3 is Positive - should say true', isPositive(3) );
+console.log( 'The number 0 is Positive - should say false', isPositive(0) );
+console.log( 'The number -3 is Positive - should say false', isPositive(-3) );
 
 
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
 function getLast( array ) {
-
+let lastItem = array[array.length - 1];
+return lastItem;
```

Comment:
Nice simple implementation. A lot of people missed that you can just return this value directly. You could go even further and remove the variable if you wanted.

```js
function getLast(array) {
  return array[array.length - 1];
}
```